src/main/python/rlbot/setup_manager.py

```diff
@@ -29,6 +29,22 @@
 
 
 class SetupManager:
```

Comment:
I definitely need to split `run()` into `start_match()` and `infinite_loop()`, otherwise I can't have the control flow necessary for training to work. i.e. `start_match()` gets called and then we loop until the test is passed; and I'd like to change match without needing to re-launch quick-chat. Also this gives us the opportunity to make naming consistent: now everything which deals with subprocesses is named `launch_foo` whereas previously there was a mix of `init_` and `launch_`.

I'd rather visibly change the API rather than silently change the definition of what `run()` does which could lead to unexpected behavior.

Maybe we should have a higher-level public API which just does everything to run a game given a config_path but I think that should be outside this class.

Reply:
Your philosophy sounds good, and I agree that it's better this way. @IamEld3st and @hallo-doei you have a short window of time to complain before we go ahead and do this :)